spec/lib/whitehall_importer/create_revision_spec.rb

```diff
@@ -69,12 +70,21 @@
             build(:whitehall_export_image, filename: "subdir/image.jpg"),
           ],
         )
-        revision = described_class.call(document, whitehall_edition)
+        revision = described_class.call(document_import, whitehall_edition)
 
         expect(revision.image_revisions.first.blob_revision.filename).to eq("image.jpg")
         expect(revision.image_revisions.last.blob_revision.filename).to eq("image-1.jpg")
       end
 
+      it "skips any image it has encountered before" do
+        image = build(:whitehall_export_image, filename: "image.jpg")
+        revision1 = described_class.call(document_import, build(:whitehall_export_edition, images: [image]))
+        revision2 = described_class.call(document_import, build(:whitehall_export_edition, images: [image]))
+        expect(revision1.image_revisions.count).to eq(1)
+        expect(revision2.image_revisions.count).to eq(1)
+        expect(revision1.image_revisions.first).to eq(revision2.image_revisions.first)
```

Comment:
Can we assert here that `expect(revision1.image_revisions).to eq(revision2.image_revisions)`?